pkg/agent/route/route_linux.go

```diff
@@ -1245,6 +1258,38 @@ func (c *Client) initWireguard() {
 	}
 }
 
+func (c *Client) initNodeLatency() {
+	gateway := "antrea-gw0"
+	if c.networkConfig.TrafficEncapMode.String() == "networkPolicyOnly" {
+		gateway = "transport"
```

Comment:
this is not correct, there is no interface names `transport`. But there is a `TransportIface` field in the `networkConfig`.